src/core/lib/surface/call_internal.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * Copyright 2015 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_CORE_LIB_SURFACE_CALL_INTERNAL_H
+#define GRPC_CORE_LIB_SURFACE_CALL_INTERNAL_H
+
+/* gRPC internal version (i.e., does not create an exec_ctx) of
+ * grpc_call_cancel. */
+void grpc_call_cancel_internal(grpc_call* call);
```

Comment:
There's no need to create a new header file for this function; just declare it in the existing call.h file.

Reply:
Done.